### ISSUE TITLE
Formula 8 Consulting

### ISSUE THREAD

**Issue by Edisoneh** [zhizdev/sparsefusion#22]
Hello, author
It's a great job, but I have some questions about the code and look forward to your answers.
1.I didn't find the perceptual loss training mentioned in formula 8 of the paper. In "vldm.py" line 621 only has l2 loss. 
2.My understanding is that both the pred object and the target object should be images, corresponding to formula 8 of the paper, whether the pred object needs to be gradient truncated.
thank you!

**Comment by Edisoneh** [comment]
In Q2:
In "train.py" line 386
with torch.no_grad():
    images_z = vae.encode(normalize(batch_rgb)).mode() * args.z_scale_factor
You are still training diffusion model in training stage.
